src/SFA.DAS.EmployerAccounts.UnitTests/Queries/GetUserByEmailTests/WhenIGetAUser.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Threading.Tasks;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.EmployerAccounts.Data;
+using SFA.DAS.EmployerAccounts.Interfaces;
+using SFA.DAS.EmployerAccounts.Models.UserProfile;
+using SFA.DAS.EmployerAccounts.Queries.GetUserByEmail;
+using SFA.DAS.NLog.Logger;
+using SFA.DAS.Validation;
+
+namespace SFA.DAS.EmployerAccounts.UnitTests.Queries.GetUserByEmailTests
+{
+    public class WhenIGetAUser : QueryBaseTest<GetUserByEmailQueryHandler, GetUserByEmailQuery, GetUserByEmailResponse>
+    {
+        private Mock<IUserAccountRepository> _repository;
+        private User _user;
+
+        public override GetUserByEmailQuery Query { get; set; }
+        public override GetUserByEmailQueryHandler RequestHandler { get; set; }
+        public override Mock<IValidator<GetUserByEmailQuery>> RequestValidator { get; set; }
+
+        [SetUp]
+        public void Arrange()
+        {
+            base.SetUp();
+
+            _user = new User();
+
+            _repository = new Mock<IUserAccountRepository>();
+
+            _repository.Setup(x => x.Get(It.IsAny<string>())).ReturnsAsync(_user);
+
+            Query = new GetUserByEmailQuery { Email = "test@test.com" };
+            RequestHandler = new GetUserByEmailQueryHandler(_repository.Object, RequestValidator.Object);
+        }
+
+        [Test]
+        public async Task ThenIShouldThrowExceptionIfTheUserCannotBeFound()
```

Comment:
Is the test title right? as it returns null rather than throws? or is the test meant to be for the validator where an email isn't passed and so throws InvalidRequest?